### DISCUSSION TITLE
How to use HTML in text within Quarto?

### DISCUSSION THREAD

**Discussion by Hedonical** [quarto-dev/quarto-cli#1969] [Q&A] [answered]
Hi everyone,

I am new to quarto/html so forgive me if this is a basic question. I want to use html containers within my text, but I haven't found an example of how to do this that is properly rendered.

Take for example the code below:
![htmlnotrendered](https://user-images.githubusercontent.com/91704211/185441161-18cea405-91e4-4bdc-a02d-88cd21db9e86.PNG)

Yes, I know that this can be done using the default settings in quarto, but I would like to use plain html to I can have better fine tune control. My main question is why this is not rendered when I click the live preview button (see image below).

![htmlrendered](https://user-images.githubusercontent.com/91704211/185441634-1a7af979-a99c-4450-a16a-916239c536ff.PNG)

I would expect it just to appear as a centered heading with the proper color. Here are my doc settings:

![docsettings](https://user-images.githubusercontent.com/91704211/185442359-4eb95ea7-a208-45b5-a5b4-26c7fa174519.PNG)

Thank you so much ahead of time!

**Comment by dragonstyle** [comment]
Your 'html' key should be indented under 'format' (remove the pdf, which is causing this to render as a pdf).

**Reply by Hedonical** [reply]
Thanks so much for the response! I removed the pdf part but it still seems to not be properly rendered
![visualedited](https://user-images.githubusercontent.com/91704211/185444854-4238461d-1532-4c4e-8f49-fa68c0c4639e.PNG)
.

**Comment by jjallaire** [comment]
The `<html>` tag is something that encloses an entire document as opposed to something you use within a document. In Quarto you can include raw HTML with e.g.

````markdown
```{=html}
<center>
  Text
</center>
```
````

**Reply by Hedonical** [reply]
Thank you for the response! I do not want to include raw html but rather use html to render specific elements in my text.

For example instead of this:
![=htmlcodeblock](https://user-images.githubusercontent.com/91704211/185465347-57be474c-af2e-4e3b-a084-2fe6c33523c1.PNG)



I want it to look like this (using just html and not the built-in quarto setting as I am more comfortable with HTML for formatting than markdown):

![usingquartobuiltinsettings](https://user-images.githubusercontent.com/91704211/185465937-584e1530-6393-43ac-b707-ff26986c5c21.PNG)

Is there an easy way to do this?

**Comment by jjallaire** [comment, marked as answer]
```markdown
## Packages {style="color:#3365bd"}
```

or

```markdown
<h2 style="color:#3365bd">Packages</h2>
```

I think the former is vastly preferable because it will render correctly for non-HTML formats (you never know when you might need to create a non-HTML version!)

**Reply by Hedonical** [reply]
Thank you very much!
